<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="C9BD4J2" name="旅行意外险" tests="3" failures="1" timestamp="4 May 2018 02:56:12 GMT" time="27.752" errors="0">
  <testcase name="login" time="8.057" classname="test.FirstTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at test.FirstTest.login(FirstTest.java:47)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="travelMethod" time="9.206" classname="test.AnlianTravelTest"/>
  <testcase name="basic" time="12.18" classname="test.AnlianTravelTest"/>
</testsuite> <!-- 旅行意外险 -->
